### PR TITLE
Added Playback preferences and Play in VLC

### PR DESCRIPTION
**Pull request by codealchemist**
Added _"Play in VLC"_ to Playback preferences.
Saved preference remembered between app reloads.

### CODE REVIEW

**Comment by dcposch** [comment]
i like this idea. i'll rebase and test

**Comment by feross** [comment]
@codealchemist Can you rebase once more?

**Comment by codealchemist** [comment]
@feross, sure thing :)

File: bin/package.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ var all = {
 
   // Pattern which specifies which files to ignore when copying files to create the
   // package(s).
-  ignore: /^\/dist|\/(appveyor.yml|\.appveyor.yml|\.github|appdmg|AUTHORS|CONTRIBUTORS|bench|benchmark|benchmark\.js|bin|bower\.json|component\.json|coverage|doc|docs|docs\.mli|dragdrop\.min\.js|example|examples|example\.html|example\.js|externs|ipaddr\.min\.js|Makefile|min|minimist|perf|rusha|simplepeer\.min\.js|simplewebsocket\.min\.js|static\/screenshot\.png|test|tests|test\.js|tests\.js|webtorrent\.min\.js|\.[^\/]*|.*\.md|.*\.markdown)$/,
+  ignore: /^\/dist|\/(appveyor.yml|\.appveyor.yml|\.github|appdmg|AUTHORS|CONTRIBUTORS|bench|benchmark|benchmark\.js|bin|bower\.json|component\.json|coverage|doc|docs|docs\.mli|dragdrop\.min\.js|example|examples|example\.html|example\.js|externs|ipaddr\.min\.js|Makefile|min|perf|rusha|simplepeer\.min\.js|simplewebsocket\.min\.js|static\/screenshot\.png|test|tests|test\.js|tests\.js|webtorrent\.min\.js|\.[^\/]*|.*\.md|.*\.markdown)$/,
```
Comment (feross) [inline bin/package.js]: Why this change?

Comment (codealchemist) [inline bin/package.js]: Oh, I should have added comments for that one or discussed it here first, sorry.
I had issues building the app for OSX and Windows when `minimist` was removed from the package.
I will retest it.

**Comment by codealchemist** [comment]
I just tested the OSX build and I don't get any `minimist` errors.
Seems to be working ok.

**Comment by eragmus** [comment]
@feross How come this was not merged & included with version 0.10? Is something still missing? Really would be nice to have the option of VLC being used by default 👍

**Comment by feross** [comment]
Looks good to me. @codealchemist Sorry, but do you ming rebase once more? There was another option added to the Preferences page, but it shouldn't be too hard to fix the conflict.

@dcposch - this looks good to you, right?

**Comment by codealchemist** [comment]
@feross Done :)

**Comment by dcposch** [comment]
LGTM

**Comment by dcposch** [comment]
thanks for making this!

**Comment by codealchemist** [comment]
@dcposch, yw! :)

**Comment by feross** [comment]
Released in WebTorrent Desktop v0.11!

**Comment by feross** [comment]
@codealchemist Thanks for this great contribution!

I just added you as a collaborator to this repo! 🎉  Please read [CONTRIBUTING.md](https://github.com/feross/webtorrent-desktop/blob/master/CONTRIBUTING.md) if you haven't already.

Cheers! 👏 👍

**Comment by codealchemist** [comment]
Cool! My pleasure :)
Thanks @feross!
